widgetssdk/src/main/res/layout/visitor_video_floating_view.xml

```diff
@@ -2,17 +2,21 @@
 <com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This way you're inflating `MaterialCardView` inside `MaterialCardView`(FloatingVisitorVideoView), can you please try to change this `MaterialCardView` with `<merge></merge>` tag (https://developer.android.com/training/improving-layouts/reusing-layouts#Merge). I think in this case you need to move the CardView setup (bg color, corner radius,  etc.) to `FloatingVisitorVideoView`